packages/adapter-cloudflare/src/worker.js

```diff
@@ -17,7 +18,7 @@ const worker = {
 
 		let { pathname } = new URL(req.url);
 
-		// static assets
+		// immutable app assets
```

Comment:
`_app` includes non-immutable stuff like `version.json`, so for correctness it should probably be something like this

```suggestion
		// generated files
```